litestar/contrib/mako.py

```diff
@@ -121,6 +120,21 @@ def register_template_callable(
         """
         self._template_callables.append((key, template_callable))
 
+    @staticmethod
+    def render_string(template_string: str, context: Mapping[str, Any] | None = None) -> str:
+        """Render a template from a string with the given context.
+
+        Args:
+            template_string: The template string to render.
+            context: A dictionary of variables to pass to the template.
+
+        Returns:
+            The rendered template as a string.
+        """
+        context = context or {}
+        template = _MakoTemplate(template_string)
+        return template.render(**context)  # type: ignore[no-any-return]
```

Comment:
would prefer not to blanket `type ignore` these if anyone can provide some guidance

Reply:
from memory the mako lib isn't typed itself so its either this or use `cast` i think